src/Sylius/Bundle/RbacBundle/DependencyInjection/SyliusRbacExtension.php

```diff
@@ -52,20 +57,4 @@ public function load(array $config, ContainerBuilder $container)
         $container->setParameter('sylius.rbac.default_permissions', $config['permissions']);
         $container->setParameter('sylius.rbac.default_permissions_hierarchy', $config['permissions_hierarchy']);
     }
-
-    /**
-     * {@inheritdoc}
-     */
-    public function prepend(ContainerBuilder $container)
```

Comment:
That was a copy paste mistake, I copied this class from my previous PR where there was no prepend. :) Reverting! Thanks.